system-tests/test/webpack_dev_server_spec.ts

```diff
@@ -45,6 +45,24 @@ describe('@cypress/webpack-dev-server', function () {
     },
   })
 
+  systemTests.it('successfully loads and runs spec to check if relative urls are working', {
```

Comment:
Test looks good, but I think we can do one better by writing the test here: https://github.com/cypress-io/cypress/blob/develop/npm/webpack-dev-server/cypress/e2e/webpack-dev-server.cy.ts

This will give you "Cypress in Cypress" so you can actually assert the test runs, assets load, etc. What do you think? You can still use the same test project you created.

Reply:
I thought the systemTests do kind of the same, but tbh. the testing structure is kind of complex and somewhat intransparent. ^^ I'll try to move the test over :)

Reply:
Kind of - the system tests are basically testing "run" mode - like `npx cypress run`. The other tests I linked (which we call "Cypress in Cypress") will let you test against a Cypress instance running in "open" mode, as in `npx cypress open`. You can also make assertions against the Cypress instance. Ultimately, either place is probably fine for the test - if it's too much effort to move it, I wouldn't worry, but it's nice to keep the webpack tests in the `npm/webpack-dev-server` directory where possible.

Reply:
I already moved it. :)